### DISCUSSION TITLE
What is best in life?

### DISCUSSION THREAD

**Discussion by felddy** [felddy/foundryvtt-docker#137] [Q&A] [answered]
@dav3r  help me out here?  
@jsf9k how about it?

**Comment by felddy** [comment]
42

**Comment by jsf9k** [comment]
The tree of woe.

**Comment by dav3r** [comment, marked as answer]
“Get busy living or get busy dying.” — Stephen King

**Comment by felddy** [comment]
To crush your enemies, to see them driven before you, and to hear the lamentations of their ~women~ significant others.

**Comment by dav3r** [comment]
“The unexamined life is not worth living.” — Rob Cena
